src/simularium/VisGeometry.ts

```diff
@@ -1586,66 +1411,19 @@ class VisGeometry {
             // if not fiber...
             if (visType === VisTypes.ID_VIS_TYPE_DEFAULT) {
                 const response = this.getGeoForAgentType(typeId);
-
                 if (!response) {
-                    console.warn(
+                    this.logger.warn(
                         `No mesh nor pdb available for ${typeId}? Should be unreachable code`
                     );
                     return;
                 }
                 const { geometry, displayType } = response;
-                // pdb has precedence over mesh
                 if (geometry && displayType === GeometryDisplayType.PDB) {
                     const pdbEntry = geometry as PDBModel;
-                    if (this.renderStyle === RenderStyle.WEBGL1_FALLBACK) {
-                        this.legacyRenderer.addPdb(
-                            pdbEntry,
-                            visAgent,
-                            this.getColorForTypeId(typeId)
-                        );
-                    } else {
-                        if (pdbEntry !== visAgent.pdbModel) {
-                            // race condition? agents arrived after pdb did?
-                            this.resetAgentPDB(visAgent, pdbEntry);
-                        }
-                        visAgent.updatePdbTransform(1.0);
-                    }
+                    this.renderPdb(typeId, visAgent, pdbEntry);
```

Comment:
these really should be named "addPdbToDrawList" or something like that.  it's not quite rendering that happens inside there